tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -14,7 +13,11 @@ import {
 } from '../../actions/reports'
 import Button from '../Button'
 import createFileInputErrorState from '../../utils/createFileInputErrorState'
-import { handlePreview, getTargetClassName } from './utils'
```

Comment:
'Cancel' button in the file submit form: makes the form to disappear. I think it should only clear the form and not delete it.

Reply:
@reitermb @victoriaatraft hiding the form on Cancel is consistent with the other data files page, but Mo brings up a good point. thoughts?

Reply:
I'm good to move to clearing the upload portion of the form instead of hiding it. Open to doing that here or in a follow-on as long as we make sure to ticket/implement for both the TANF/SSP and FRA sides.

Reply:
I started to change the form cancel behaviors, but I forgot about a bug that prevented it from actually working

https://github.com/uswds/uswds/issues/4848

the uswds file input doesn't automatically clear the preview state when the file is set to `null`. it doesn't have a clear function exposed at all, so the only way to get the preview to disappear (even after you've cleared the file itself) is to un-render and re-render the file input. We could hide and re-show the form, but i thought the jumpiness would be poor user experience. i'll leave it the way it is for now